guides/common/modules/proc_enabling-capsule-in-UI.adoc

```diff
@@ -0,0 +1,33 @@
+[id="assigning-organization-location-capsule-server_{context}"]
+
+= Assigning the Correct Organization and Location to {SmartProxyServer} in the {Project} web UI
+
+After installing the {SmartProxyServer} packages, you must enable the {SmartProxy} in the {Project} web UI by assigning the correct organization and location to it.
```

Comment:
Is this still true? I thought https://projects.theforeman.org/issues/26092  changed it that if there's only one org (or location) that's picked automatically. The Smart Proxy is created by the installer via the API. That means `must` is only correct if there's more than one organization/location.

If my theory is correct, it should do the right thing since Foreman 1.23. IIRC that's Satellite 6.7 but I'd recommend testing on the latest version.

Reply:
Thanks @ekohl. I've added text for clarity.